docs/openapi/tools-api.yaml

```diff
@@ -109,6 +109,43 @@ import-job-status:
         $ref: './responses.yaml#/404'
       '500':
         $ref: './responses.yaml#/500'
+  patch:
+    tags:
+      - import
+    summary: Patch an import job
+    description: |
+      This endpoint can be used to stop an import job, given its jobId. This operation will prevent
+      any further scraping of the import job URLs, and mark the job status as STOPPED.
```

Comment:
Will the URLs currently being processed still have to complete?
Will any results be saved to any S3 bucket?
Will the user be able to start a new import immediately?

If you caller should know any of this, I'd include it in the description.

Reply:
Only if you think it will help the reader (and prevent them needing to ask us).  Up to you.  Maybe they will hope if they stop the job after 500 out of 1000 URLs that they would get those results... maybe?

Reply:
For now, keeping the output consistent - if the user stops a job, they do not have access to the s3 bucket.